lib/console.js

```diff
@@ -86,7 +86,12 @@ function createWriteErrorHandler(stream) {
   };
 }
 
-function write(ignoreErrors, stream, string, errorhandler) {
+function write(ignoreErrors, stream, string, errorhandler, groupIndent) {
+  if (groupIndent.length !== 0) {
```

Comment:
I'd have to move the prepending of `groupIndent` if I did that so the result would probably be:

```js
if (groupIndent.length !== 0) {
  if (string.indexOf('\n') !== -1 {
    string = string.replace(/\n/g, `\n${groupIndent}`);
  }
  string = groupIndent + string;
}
string += '\n';
```

...or...

```js
if (groupIndent.length !== 0 && string.indexOf("\n") !== -1) {
  string = string.replace(/\n/g, `\n${groupIndent}`);
}
string = groupIndent + string + '\n';
```